### DISCUSSION TITLE
Are the new "dated" stable packages a replacement for the old ones and we should switch to them?

### DISCUSSION THREAD

**Discussion by stronk7** [BerriAI/litellm#6955] [Q&A] [answered]
Sorry if this has been documented or asked anywhere, but it would be great if you can clarify which container images we should be using:

1. In the "old days" (heh, some weeks ago"), we had the following stable packages nomenclature:
   - `main-stable`, to point to the latest stable release.
   - `main-v1.52.0-stable` for each stable release.

2. Since `v1.52.0-stable`, it seems that the new nomenclature is:
  - `litellm_stable_nov27-stable`, to point to the stable release in a date.
  - `litellm_stable_nov27-v1.52.15-stable`, the same, but specifying both the date and the version.

So, my questions are:
1. Should we switch to the new nomenclature and forget about the old one? [OR]
2. Are the old ones going to keep working at some point (pointing to the same new ones). I really would **expect the git tag to match the image label too** - apart from any other "synonyms/aliases".
3. Personal opinion: Aren't those `litellm-` and `nov27` bits a little redundant, strange?
4. Is it documented anywhere? I remember that in the past I saw multiple mentions to the `main-stable` packages as being the better for prod environments, now things have become a little bit more complex.

So that's all, just a little bit confused, hence asking for clarification about all those image tags...

**Comment by ishaan-jaff** [comment, marked as answer]
> Are the new "dated" stable packages a replacement for the old ones and we should switch to them?

Yes 

> Why the change ?

We're improving our practices on deploying stable releases. We create a branch every week that gets deployed to our cloud solution and we apply critical fixes on the branch 

The stable release gets deployed from the branch - which is why the image name == the branch name
